mnc/control.py

```diff
@@ -141,7 +141,7 @@ def start_fengine(self, initialize=False, program=False, force=False):
             dests += [{'ip':dest_ip, 'port':dest_port, 'start_chan':start_chan, 'nchan':nchan}]
 
         for snap2name in snap2names:
-            f = snap2_fengine.Snap2Fengine(snap2name)
+            f = snap2_fengine.Snap2FengineEtcd(snap2name)
```

Comment:
Above my pay grade, but I wonder if it is worth exposing the choice to instantiate `Snap2FengineEtcd` rather than `Snap2Fengine` to some higher level, so that in the case of an etcd catastrophe it is easy to revert to the old method.

Similarly (probably more importantly) I think the `etcdhost` argument of the `Snap2FengineEtcd` should probably be controlled via the existing config files, in case we ever have to go to a backup server

Reply:
(Though having said that, I think the cleaner way to deal with a server change would be just to redirect the DNS)

Reply:
ditto `Contoller` should control `etcdhost`. The DNS change might be tricky since we're doing static IP and hostname assignments for some nodes.. https://github.com/ovro-lwa/lwa-issues/issues/141

Reply:
I'm adding `etcdhost` as arg and exposing `useetcd` in the `start_fengine` method.
However, `Controller` already does take an argument `etcdhost`. Default is to parse config file, but it can be overloaded with argument.